sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -157,7 +139,8 @@
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public PollerFlux<OperationResult, List<RecognizedForm>>
         beginRecognizeCustomForms(Flux<ByteBuffer> form, String modelId, long length, FormContentType formContentType) {
-        return beginRecognizeCustomForms(form, modelId, length, formContentType, false, null);
+        return beginRecognizeCustomForms(new RecognizeCustomFormsOptions(form, length, modelId)
```

Comment:
nit: It's a little confusing that RecognizeCustomFormsOptions takes arguments in the order (form, length, modelId), while this function takes it in (form, modelId, length)

Reply:
I think (form, length, modelId) could be the correct ordering. I will update to follow that ordering and get consistent, thanks!